data.sql

```diff
@@ -45,4 +45,22 @@ UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
 UPDATE animals SET owner_id = 2 WHERE name = 'Gabumon' OR name = 'Pikachu';
 UPDATE animals SET owner_id = 3 WHERE name = 'Devimon' OR name = 'Plantmon';
 UPDATE animals SET owner_id = 4 WHERE name IN ('Charmander','Squirtle','Blossom');
-UPDATE animals SET owner_id = 5 WHERE name = 'Angemon' OR name = 'Boarmon';
\ No newline at end of file
+UPDATE animals SET owner_id = 5 WHERE name = 'Angemon' OR name = 'Boarmon';
+
+/*PROJECT DAY 4 */
+INSERT INTO vets (name, age, date_of_graduation) VALUES 
+  ('William Tatcher', 45, '2000-04-23'), 
+  ('Maisy Smith', 26, '2019-01-17'), 
+  ('Stephanie Mendez', 64, '1981-05-04'), 
+  ('Jack Harkness', 38, '2008-01-08');
+
+  INSERT INTO specializations (vet_id,species_id) VALUES (1,1), (3,1),(4,2);
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/230601765-19701ef8-91b0-467f-bd8b-0796911fcc42.png)

- [ ] It would be nice if you add specialization of `Stephanie Mendez` with `Digimon` as well 👍 to follow the requirements 👍